src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const initialState = [];
+
+export const checkStatus = () => ({
+  type: 'CHECK_STATUS',
+});
+
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'CHECK_STATUS':
+      return [...state, 'IN_PROGRESS'];
```

Comment:
- [ ] Kindly check on your returned string, it's supposed to return the "Under construction" string, this is a project requirement.
![image](https://user-images.githubusercontent.com/61408860/200666816-8ad09758-8e78-4f49-a4b7-15f27d0e69b8.png)